tests/acceptance/features/apiGraphUserGroup/getUser.feature

```diff
@@ -1437,160 +1438,236 @@ Feature: get users
+                "id",
+                "userType"
+              ],
+              "properties": {
+                "displayName": {
+                  "type": "string",
+                  "const": "Alice Hansen"
+                },
+                "id": {
+                  "type": "string",
+                  "pattern": "^%user_id_pattern%$"
+                },
+                "userType": {
+                  "type": "string",
+                  "const": "Member"
+                }
               }
             }
           }
         }
       }
-    }
-    """
+      """
 
 
   Scenario: non-admin user searches for a disabled users
     Given the user "Admin" has disabled user "Alice"
     When user "Brian" searches for user "alice" using Graph API
     Then the HTTP status code should be "200"
+    And the JSON data of the search response should not contain user email
     And the JSON data of the response should match
-    """
-    {
-      "type": "object",
-      "required": [
-        "value"
-      ],
-      "properties": {
-        "value": {
-          "type": "array",
-          "minItems": 1,
-          "maxItems": 1,
-          "items": {
-            "type": "object",
-            "required": [
-              "displayName",
-              "id",
-              "userType"
-            ],
-            "properties": {
-              "displayName": {
-                "type": "string",
-                "enum": ["Alice Hansen"]
-              },
-              "id": {
-                "type": "string",
-                "pattern": "^%user_id_pattern%$"
-              },
-              "userType": {
-                "type": "string",
-                "enum": ["Member"]
+      """
+      {
+        "type": "object",
+        "required": [
+          "value"
+        ],
+        "properties": {
+          "value": {
+            "type": "array",
+            "minItems": 1,
+            "maxItems": 1,
+            "items": {
+              "type": "object",
+              "required": [
+                "displayName",
+                "id",
+                "userType"
+              ],
+              "properties": {
+                "displayName": {
+                  "type": "string",
+                  "const": "Alice Hansen"
+                },
+                "id": {
+                  "type": "string",
+                  "pattern": "^%user_id_pattern%$"
+                },
+                "userType": {
+                  "type": "string",
+                  "const": "Member"
+                }
               }
             }
           }
         }
       }
-    }
-    """
+      """
 
 
   Scenario: non-admin user searches for multiple users having same displayname
     Given the user "Admin" has created a new user with the following attributes:
       | userName    | another-alice                |
-      | displayName | Alice Murphy                 |
+      | displayName | Alice Hansen                 |
```

Comment:
The purpose here was to have two users `Alice Hansen` and `Alice Murphy` then Brian searches for `Alice` which gives two Alice entries
```suggestion
      | displayName | Alice Hansen                 |
```

Reply:
the title of this scenario is `non-admin user searches for multiple users having same displayname` and I thought we should have two `Alice Hansen`  users. And we found two `Alice Hansen`  one is - with user role, second with admin role. 

Now I see that it's correct. I suggest to leave as it

Reply:
>And we found two Alice Hansen one is - with user role, second with admin role.

:+1: Yeah, that should be separate the two